.github/issue_template.md

```diff
@@ -70,15 +63,15 @@ Nextcloud version:
 Storage backend (external storage):
 
 ### Logs
+<!-- desktop client logs are a hard requirement for bug reports because we don't know how to do magic here :) -->
 
 Please use Gist (https://gist.github.com/) or a similar code paster for longer
 logs.
 
 1. Client logfile: 
-<!-- desktop client logs are a hard requirement for bug reports because we don't know how to do magic here :) -->
-Output of `nextcloud --logdebug --logwindow` or `nextcloud --logdebug --logfile log.txt`
-(On Windows using `cmd.exe`, you might need to first `cd` into the Nextcloud directory)
-(See also https://docs.nextcloud.com/desktop/2.3/troubleshooting.html#log-files)
+Under the "General" settings, you can click on "Create Debug Archive ..." to pick the location of where the desktop client will export the logs and the database to a zip file.
```

Comment:
Maybe mention this is for 3.1 and superior, and that for older releases the command line flags are the way to do?

Reply:
I fixed it. Is that clear now?

Reply:
Yes. Thanks.